src/generate-array.js

```diff
@@ -0,0 +1,19 @@
+const exec = require('child_process').exec
+
+module.exports = (command, callback) => {
+  exec(command, (err, text, stderr) => {
+    if (err) {
+      console.error(`Harmless Changes: error when reading ${command}. See logs below.`)
+      console.error(err.stack)
+      console.error(text.toString())
+      console.error(stderr.toString())
+      process.exit(1)
+    } else {
+      // convert .ciignore to Array
+      const arr = text.split(/\n/)
+      // remove extra item
+      arr.pop()
```

Comment:
What extra item? If you're trying to get rid of an empty item in the array, do a filter on your split instead

Reply:
`text.split` turns the code into an array so `arr.pop()` does what is required.